docs/architecture/adr-002-ibc-relayer.md

```diff
@@ -4,19 +4,27 @@
 * {date}: {changelog}
 
 ## Definitions
+These definitions are specific for this document and they may not be consistent with the IBC Specification.
+
+IBC datagram - the transaction payload sent by the relayer over the physical network that includes client, connection, channel and IBC packet data. 
+
+IBC packet - a particular type of IBC datagram that includes the application packet and its commitment proof.
+
+IBC datagram transaction - a transaction that includes an IBC datagram.
```

Comment:
(This suggestion is rather a test to see if I understood what you mean.)
```suggestion
IBC datagram transaction - an on-chain transaction that includes an IBC datagram.
```

Reply:
Why on-chain? We talk about this before it is submitted to the destination chain